.appveyor.yml

```diff
@@ -38,11 +38,11 @@ init:
 
 build_script:
   - ctest -VV -S cmake/ctest/script_ci.ctest
+  - msbuild %APPVEYOR_BUILD_FOLDER%\build\rwgame\INSTALL.vcxproj
```

Comment:
I believe calling `msbuild` explicitly can be changed to 
```
cmake --target install --config Release "%APPVEYOR_BUILD_FOLDER%\build"
```
(or something similar).
That would make it build system agnostic.

Reply:
We want to install all or "only" rwgame?